### ISSUE TITLE
Add a validation suite for running after updates

### ISSUE THREAD

**Issue by cwwalter** [WCSim/WCSim#10]
We should add a set of validation input files and .mac files so that we can check after updates that things still work properly. 

This might be in a separate repository.

**Comment by tdealtry** [comment]
I was just looking at my assigned issues, and saw this (related to #162). From what I understand, @brichards64 has got Travis CI working, and it just needs someone with WCSim admin privilege to turn on.
This will be very useful in the near future (i.e. when #178 is merged) so that the other pending (and all future) pull requests don't have to be revalidated by authors. Instead this can be done automatically by Travis CI.
Of course after #162 is solved, we can leave this open until the exact physics checks are defined.

**Comment by eosullivan** [comment]
Ok, so let's turn this on so we can at least start playing around with it a bit more. @brichards64, can you let me know what you need me to do to make this live? I also had some future improvement suggestions in our last email exchange, which would be a nice to have. I will attach them below:

 For future improvement, I can probably think of a couple other tests that would be nice for WCSim. A test to make sure a pull request is not significantly increasing the time to generate events would be great. And we also probably want to test with various detector configurations (SK, the HK options, etc) in case a change has been made to a specific detector component (if someone makes a change to the 20" B&L, for example).

**Comment by brichards64** [comment]
Sure, Its been a long time since i made it and i think wcsim has advanced since then so ill check it all still works and make any changes needed then ill let you know how to integrate it.

From memory requires an admin to link the github account with the build server and add a couple of scripts. If you want validation as well as build testing adding a webpage to github is the best way to go. But ill outline the steps needed when i check its all still works well. I might also make some improvements to the validation such that anyone can add validation of any quantity they want easily.
